### PR TITLE
feat: update dynamic widget summary type, add showing top label to dynamic chart and update icons

### PR DESCRIPTION
**Pull request by WANZARGEN**
## To Reviewers
- [ ] Skipping reviews is not a problem

## Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
  - [x] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Dynamic Widget 

- Added "showing top" label
   - Display label when data length of `chart` type widget is larger than limit
- Changed the type name (`chart` -> `summary`)
- Updated `summary` type widget design
- Changed `summary` type widget's loader to be reflected only in value area
- Added "using widget list" story for describe `index` prop

### Dynamic Chart

- Added `limit` prop

### Icon
- Added new icons
- Changed usages of icon name in components (ic_working -> ic_loading)
  - [x] This change require a usage change

### CODE REVIEW

File: src/data-display/dynamic/dynamic-widget/templates/chart/index.vue [hunk 1]
```diff
@@ -76,6 +78,17 @@ export default defineComponent<DynamicWidgetChartProps>({
                 const themeIdx = props.index % DYNAMIC_CHART_THEMES.length;
                 return DYNAMIC_CHART_THEMES[themeIdx];
             }),
+            chartData: computed<any[]>(() => {
+                if (Array.isArray(props.data)) return props.data;
+                if (props.data === undefined || props.data === null) return [];
+                return [props.data];
+            }),
+            chartType: computed<DynamicChartType>(() => props.schemaOptions.chart_type || 'COLUMN'),
+            chartLimit: computed<number|undefined>(() => DYNAMIC_CHART_LIMIT_MAP[state.chartType]),
```
Comment (siyeons) [inline src/data-display/dynamic/dynamic-widget/templates/chart/index.vue:87]: 궁금한게 있습니다! 위에서는 props.limit의 undefined를 체크하지 않고 limit이 무조건 있을 경우로 작성되어 있는 것 같은데 
상관없을까요??? 

```
            themeColorSetMap: computed<Record<DynamicChartTheme, string[]>>(() => ({
                VIOLET: getColorSet(0, props.limit),
                BLUE: getColorSet(1, props.limit),
                CORAL: getColorSet(2, props.limit),
(생략)
```

**Review by siyeons — APPROVED** [review]
LGTM!!

**Review by JeoungSulMo — COMMENTED** [review]
LGTM입니당

File: src/data-display/dynamic/dynamic-widget/templates/chart/index.vue (hunk 1 above)
Comment (WANZARGEN) [reply src/data-display/dynamic/dynamic-widget/templates/chart/index.vue:87]: 엇 추가할게용

**Pull request merged**